charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -77,8 +77,8 @@ insecureScheme:
 config:
   logLevel: info
   domains:
-    - localhost
     - oidc-discovery.example.org
+    - localhost
```

Comment:
Why was this swap of domains required?

Reply:
The test is only testing the first entry in the list... localhost wont resolve to anything useful within the test pod cause it has its own 127.0.0.1 separate from the discovery pod. localhost being in the list is somewhat useful for testing the helm chart within minikube using port-forwarding.